lib/Server.js

```diff
@@ -1133,13 +1133,15 @@ class Server {
 
           // cert is more than 30 days old, kill it with fire
           if ((now - Number(certificateStat.ctime)) / certificateTtl > 30) {
```

Comment:
This is the only production file that uses `del`, the rest of the usages are all in tests.